lib/utils/npm-exists.js

```diff
@@ -0,0 +1,18 @@
+const got = require('got');
+
+const npmSource = {
+	taobao: 'https://npm.taobao.org',
+	npm: 'https://www.npmjs.org'
+};
+
+const constant = value => () => value;
```

Comment:
In that case, if we care using the source code of `npm-exists` then we should refer to it as a dependency and install it via npm, instead of copy paste the code into our project